### PR TITLE
Update Kubernetes Agent readme with Octopus Deploy version

### PR DESCRIPTION
**Pull request by APErebus**
Just to give more context about what version of Octopus Server supports this version of the Agent

### CODE REVIEW

File: charts/kubernetes-agent/README.md.gotmpl [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 
 {{ template "chart.maintainersSection" . }}
 
 {{ template "chart.sourcesSection" . }}
 
-{{ template "chart.valuesSection" . }}
\ No newline at end of file
+{{ template "chart.valuesSection" . }}
+
+{{ template "helm-docs.versionFooter" . }}
```
Comment (liam-mackie) [inline charts/kubernetes-agent/README.md.gotmpl:16]: Is there a reason we're adding the `helm-docs` footer in?

Comment (APErebus) [reply charts/kubernetes-agent/README.md.gotmpl:16]: Yeah. I think it's appropriate that we acknowledge the use of this tool via the footer. Give readers information that they might want to use the tool as well

Comment (liam-mackie) [reply charts/kubernetes-agent/README.md.gotmpl:16]: The only reason I didn't was to stop the version from showing diffs when the container updates on a dev machine vs. CI, not to hide the fact we use the tool 😅

Comment (APErebus) [reply charts/kubernetes-agent/README.md.gotmpl:16]: oo, fair... I'll remove it this and replace with with a fixed footer that won't change

**Review by liam-mackie — APPROVED** [review]
LGTM! Thanks for the cleanup!

**Pull request merged**